content/docs/en/getting-started/1-quick-start.md

```diff
@@ -9,19 +9,22 @@ it in the [NativeScript Playground](https://play.nativescript.org?template=play-
 If you are ready to move on and work locally on your machine, and [you
 have your system ready for native development](/en/docs/getting-started/installation), you can start by using a template:
 
-```shell
-$ tns create my-app --template nativescript-vue-template
-$ cd my-app
+###  Scaffold project
+```sh
+$ npm install -g vue-cli
```

Comment:
We should probably start advising vue-cli 3.x (`@vue/cli` with `@vue/cli-init`)

Reply:
Not following you entirely, should I change the command or just suggest that this is the we would do it in vue-cli 3?

Reply:
Well, I think vue cli 3.x will be the default very soon, so we might as well make the users install it right off the bat. 

If both `@vue/cli` and `@vue/cli-init` are installed globally, we can use the 2.x templates like normal (`vue init nativescript-vue/vue-cli-template <ProjectName>`).

So just change this line to 
```
$ npm install -g @vue/cli @vue/cli-init
```

Reply:
updated